Docs/user_guide/quantization_configuration.rst

```diff
@@ -39,53 +40,53 @@ When working with a new runtime with different rules, or for experimental purpos
 
         - is_symmetric:
-            An optional parameter. If included, possible settings include "True" and "False".
-            A "True" setting will place all parameter quantizers in symmetric mode by default.
-            A "False" setting, or omitting the parameter altogether, will set all parameter quantizers to asymmetric mode by default.
+            Optional.  If included, value is "True" or "False".
+            "True" places all parameter quantizers in symmetric mode by default.
+            "False", or omitting the parameter, sets all parameter quantizers to asymmetric mode by default.
 
-    Aside from the "ops" and "params" dictionary, additional settings governing quantizers in the model are available:
+    Outside the "ops" and "params" dictionaries, the following additional quantizer settings are available:
 
     - strict_symmetric:
-        An optional parameter. If included, possible settings include "True" and "False".
-        When set to "True", quantizers which are configured in symmetric mode will use strict symmetric quantization.
-        When set to "False" or omitting the parameter altogether, quantizers which are configured in symmetric mode will not use strict symmetric quantization.
+        Optional.  If included, value is "True" or "False".
+        "True" causes quantizers configured in symmetric mode to use strict symmetric quantization.
+       "False", or omitting the parameter, causes quantizers configured in symmetric mode to not use strict symmetric quantization.
 
     - unsigned_symmetric:
-        An optional parameter. If included, possible settings include "True" and "False".
-        When set to "True", quantizers which are configured in symmetric mode will use unsigned symmetric quantization when available.
-        When set to "False" or omitting the parameter altogether, quantizers which are configured in symmetric mode will not use unsigned symmetric quantization.
+        Optional.  If included, value is "True" or "False".
+        "True" causes quantizers configured in symmetric mode use unsigned symmetric quantization when available.
+       "False", or omitting the parameter, causes quantizers configured in symmetric mode to not use unsigned symmetric quantization.
 
     - per_channel_quantization:
-        An optional parameter. If included, possible settings include "True" and "False".
-        When set to "True", parameter quantizers will use per channel quantization as opposed to per tensor quantization.
-        When set to "False" or omitting the parameter altogether, parameter quantizers will use per tensor quantization.
+        Optional.  If included, value is "True" or "False".
+        "True" causes parameter quantizers to use per-channel quantization rather than per-tensor quantization.
+        When set to "False" or omitting the parameter, causes parameter quantizers to use per-tensor quantization.
```

Comment:
Minor: Can make the phrasing here consistent with the other flags

Reply:
Fixed throughout the file.